pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -572,6 +573,65 @@ class ClassTypeHintMatch:
     ),
 ]
 _INHERITANCE_MATCH: dict[str, list[ClassTypeHintMatch]] = {
+    "air_quality": [
+        ClassTypeHintMatch(
+            base_class="Entity",
+            matches=_ENTITY_MATCH,
+        ),
+        ClassTypeHintMatch(
+            base_class="AirQualityEntity",
+            matches=[
+                TypeHintMatch(
+                    function_name="device_class",
+                    return_type=["str", None],
+                ),
+                TypeHintMatch(
+                    function_name="particulate_matter_2_5",
+                    return_type=STATE_TYPE,
```

Comment:
Should we really accept strings for all these? or should it be adjusted to [None, "int", "float"] ?